osier/models/capacity_expansion.py

```diff
@@ -7,12 +7,17 @@
 import functools
 import time
 
-from osier import DispatchModel
+from osier import DispatchModel, LogicDispatchModel
 
 from pymoo.core.problem import ElementwiseProblem
 
 LARGE_NUMBER = 1e20
 
+dispatch_models = {'lp': DispatchModel,
+                   'hierarchical': LogicDispatchModel,
+                   'logical': LogicDispatchModel,
+                   'rule_based': LogicDispatchModel}
```

Comment:
`lp` feels a bit too brief. Consider making it more descriptive. And why are there 3 keys for selecting `LogicDispatchModel`? Consider condensing those into one key.